cmd/internal/root.go

```diff
@@ -62,13 +62,15 @@ var rootCmd = &cobra.Command{
 
 		host := viper.GetString("operator-webhook-service-host")
 		port := viper.GetInt32("operator-webhook-service-port")
+		provider := viper.GetString("provider")
 
-		if host == "" {
+		if host == "" && provider == "" {
```

Comment:
nit: better error message for empty provider

Reply:
It is actually not for error message. `provider` is optional. host is necessary if provider is not given. If provider is given, then host becomes optional as we use service address for webhook.